Sprint-2/implement/tally.js

```diff
@@ -1,3 +1,30 @@
-function tally() {}
+function tally(inputArray) {
+    //initialise an empty object
+    let talliedResults = {};
+    //check if input is an array
+    if (Array.isArray(inputArray)) {
+        //for an empty array
+        if (inputArray.length === 0) {
+            return talliedResults;
+        }
 
+        for (const property of inputArray) {
+            //if this propert is already in the object increment it
+            if (talliedResults[property]) {
```

Comment:
The solution works but can be simplified with single statement of keeping if else in a loop. Please try a simpler version.

Reply:
I've implemented a simpler solution, please let me know if anything else is needed @vamsikrishna-oc